runtime/compiler/build/toolcfg/zos-xlc/common.mk

```diff
@@ -223,13 +225,16 @@ AR_CMD?=$(AR)
 SOLINK_CMD?=$(CXX)
 
 SOLINK_FLAGS+=-g -Wl,xplink,dll,map,list,compat=$(TGTLEVEL)
-SOLINK_LIBPATH+=$(J9SRC) $(J9SRC)/lib
-SOLINK_SLINK+=$(PRODUCT_SLINK)
+SOLINK_LIBPATH+=$(J9SRC) \
+                $(J9SRC)/lib \
+                $(J9SRC)/zos_zlib/hzc/lib \
+                /usr/lpp/hzc/lib
```

Comment:
Just to clarify for others, is the intention here to take advantage of include precedence to look for the `zz` library in `$(J9SRC)/zos_zlib/hzc/lib` first, and if it doesn't exist there to look for the system library in `/usr/lpp/hzc/lib` which is the default ZLIB path on z/OS?

Reply:
Yes. On zOS , we have a `libzz.a` located at `/usr/lpp/hzc/lib` which is fallback path if it can not locate in `$(J9SRC)/zos_zlib/hzc/lib`